server/aws-lsp-codewhisperer/src/language-server/agenticChat/context/agenticChatTriggerContext.ts

```diff
@@ -119,7 +119,7 @@ export class AgenticChatTriggerContext {
         }
 
         if (hasWorkspace) {
-            promptContent = promptContent?.replace(/^@workspace\/?/, '')
+            promptContent = promptContent?.replace(/\*\*@workspace\*\*/, '')
```

Comment:
nit: it's good to replace both options - bold and not bold, otherwise we depend on a very specific UI here